### PR TITLE
Reduction_plan listing

### PR DESCRIPTION
**Pull request by alvineteubo**
[6](https://github.com/uni2growcm/u2g-openhospital-gui/issues/6)

### CODE REVIEW

File: bundle/language_en.properties [hunk 1]
```diff
@@ -1517,6 +1520,16 @@ angal.pricesothers.othm
 angal.pricesothers.pleaseselectanitemtodelete                                                          = Please select an item to delete
 angal.pricesothers.pleaseselectanitemtoedit                                                            = Please select an item to edit
 angal.pricesothers.thedatacouldnotbedeleted                                                            = The data could not be deleted
+angal.reduction.exam                                                                                   = Exam
+angal.reduction.examrate.col                                                                           = Exam rate
+angal.reduction.medical                                                                                = Medical
+angal.reduction.medicalrate.col                                                                        = Medical rate
+angal.reduction.operate.col                                                                            = Ope. rate
```
Comment (JantBogard) [inline bundle/language_en.properties]: Why did you write `Ope. rate` like this?

File: bundle/language_fr.properties [hunk 2]
```diff
@@ -1058,6 +1059,7 @@ angal.menu.btn.pharmacy
 angal.menu.btn.pretreatmenttype                                                                        = Typologie Traitements Grossesse
 angal.menu.btn.priceslists                                                                             = Liste des prix
 angal.menu.btn.printing                                                                                = Rapports
+angal.menu.btn.reductionplan                                                                           = Plan de Reduction
```
Comment (JantBogard) [inline bundle/language_fr.properties]:
```suggestion
angal.menu.btn.reductionplan                                                                           = Plan de Réduction
```

File: bundle/language_fr.properties [hunk 3]
```diff
@@ -1116,6 +1118,7 @@ angal.menu.pharmacy
 angal.menu.pretreatmenttype                                                                            = Typologie Traitements Grossesse
 angal.menu.priceslists                                                                                 = Liste des prix
 angal.menu.printing                                                                                    = Impression
+angal.menu.reductionplan                                                                               = Plan de reduction
```
Comment (JantBogard) [inline bundle/language_fr.properties]:
```suggestion
angal.menu.reductionplan                                                                               = Plan de réduction
```

File: bundle/language_fr.properties [hunk 4]
```diff
@@ -1507,6 +1510,16 @@ angal.pricesothers.othm
 angal.pricesothers.pleaseselectanitemtodelete                                                          = Veuillez sélectionner un élément à supprimer
 angal.pricesothers.pleaseselectanitemtoedit                                                            = Sélectionner un élément à modifier
 angal.pricesothers.thedatacouldnotbedeleted                                                            = La date ne peut pas être supprimée
+angal.reduction.exam                                                                                   = Examen
+angal.reduction.examrate.col                                                                           = Taux examen.
+angal.reduction.medical                                                                                = M\u00E9dical
```
Comment (JantBogard) [inline bundle/language_fr.properties]:
```suggestion
angal.reduction.medical                                                                                = Médical
```
Do the same thing on the other lines.

**Review by JantBogard — CHANGES_REQUESTED** [review]

File: bundle/language_en.properties (hunk 1 above)
Comment (alvineteubo) [reply bundle/language_en.properties]: it's how it is writhing in oh_dschang. will i change it???

File: bundle/language_fr.properties [hunk 5]
```diff
@@ -1510,15 +1510,15 @@ angal.pricesothers.othm
 angal.pricesothers.pleaseselectanitemtodelete                                                          = Veuillez sélectionner un élément à supprimer
 angal.pricesothers.pleaseselectanitemtoedit                                                            = Sélectionner un élément à modifier
 angal.pricesothers.thedatacouldnotbedeleted                                                            = La date ne peut pas être supprimée
-angal.reduction.exam                                                                                   = Examen
+angal.reduction.exam                                                                                   = Examen.
```
Comment (JantBogard) [inline bundle/language_fr.properties]:
```suggestion
angal.reductionplan.exam.label                                                                                   = Examen
```

File: bundle/language_fr.properties [hunk 6]
```diff
@@ -1510,15 +1510,15 @@ angal.pricesothers.othm
...
-angal.reduction.otherrate.col                                                                          = Autres Taux
-angal.reduction.reductionrate                                                                          = Taux de r\u00E9duction
+angal.reduction.medical                                                                                = Medical.
+angal.reduction.medicalrate.col                                                                        = Taux medicament.
+angal.reduction.operate.col                                                                            = Taux operation.
+angal.reduction.operation                                                                              = Operation.
+angal.reduction.other                                                                                  = Autres services.
+angal.reduction.otherrate.col                                                                          = Autres Taux.
+angal.reduction.reductionrate                                                                          = Taux de reduction.
```
Comment (JantBogard) [inline bundle/language_fr.properties]:
```suggestion
angal.reductionplan.medical.label                                                                       = Médical
angal.reductionplan.medicalrate.col                                                                    = Taux médicament
angal.reductionplan.operate.col                                                                        = Taux opération
angal.reductionplan.operation                                                                          = Opération
angal.reductionplan.other                                                                              = Autres services
angal.reductionplan.otherrate.col                                                                      = Autres Taux
angal.reductionplan.reductionrate                                                                      = Taux de réduction
```

**Review by JantBogard — CHANGES_REQUESTED** [review]

**Review by ArnaudFofou — APPROVED** [review]

**Pull request closed without merging**